### ISSUE TITLE
kubebox 0.6.0 segmentation fault on Linux

### ISSUE THREAD

**Issue by malcolm061990** [astefanutti/kubebox#58]
I updated to the last kubebox 0.6.0 version.

Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.3 LTS
Release:	18.04
Codename:	bionic

After I start the kubebox it crashes nearly in 10 seconds with the error:
`Segmentation fault (core dumped)`

It makes the soft unusable.

The **strace** log:

```
sudo strace -p 11043
strace: Process 11043 attached
strace: [ Process PID=11043 runs in x32 mode. ]
strace: [ Process PID=11043 runs in 64 bit mode. ]
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\207\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
futex(0x7f9e56cb9a04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\217\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 72, NULL, 8)   = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_WRITE) = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_EXEC) = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\213\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\231\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 77, NULL, 8)   = 0
futex(0x7f9e574bca04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\271\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 74, NULL, 8)   = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_WRITE) = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_EXEC) = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\270\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 73, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\274\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 73, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\264\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\246\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 78, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\247\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\207\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\217\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\213\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\231\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 74, NULL, 8)   = 0
futex(0x7f9e564b6a04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e57cbfa04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e56cb9a04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e574bca04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e564b6a04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e57cbfa04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
getpid()                                = 11043
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\271\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
futex(0x7f9e56cb9a04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\270\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_WRITE) = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_EXEC) = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\274\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 77, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\264\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
futex(0x7f9e574bca04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
mprotect(0x662d2804000, 503808, PROT_READ|PROT_WRITE) = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_EXEC) = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\246\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 72, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\247\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\207\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\217\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 74, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\213\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 73, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\231\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\271\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\270\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 73, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\274\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 77, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\264\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\246\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\247\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 78, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\207\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\217\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\213\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\231\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\271\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 73, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\270\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 74, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\274\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 73, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\264\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 77, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\246\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
futex(0x7f9e564b6a04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e57cbfa04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e56cb9a04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e574bca04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e564b6a04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e57cbfa04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
getpid()                                = 11043
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\247\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 73, NULL, 8)   = 0
mmap(NULL, 290816, PROT_READ|PROT_WRITE, MAP_PRIVATE|MAP_ANONYMOUS, -1, 0) = 0x7f9e57d94000
futex(0x7f9e56cb9a04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\207\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 69, NULL, 8)   = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_WRITE) = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_EXEC) = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\217\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 74, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\213\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 74, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\231\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\271\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\270\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\274\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 74, NULL, 8)   = 0
futex(0x7f9e574bca04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_WRITE) = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_EXEC) = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\264\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 74, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\246\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\247\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 76, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\207\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 77, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\217\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 77, NULL, 8)   = 0
futex(0x7f9e564b6a04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
mprotect(0x662d2804000, 503808, PROT_READ|PROT_WRITE) = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_EXEC) = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\213\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\231\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 74, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\271\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 75, NULL, 8)   = 0
futex(0x7f9e57cbfa04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
mprotect(0x662d2804000, 503808, PROT_READ|PROT_WRITE) = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_EXEC) = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\270\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 74, NULL, 8)   = 0
futex(0x7f9e56cb9a04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 1
mprotect(0x662d2804000, 503808, PROT_READ|PROT_WRITE) = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_EXEC) = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\274\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 73, NULL, 8)   = 0
futex(0x7f9e574bca04, FUTEX_WAKE_PRIVATE, 1) = 1
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\264\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 72, NULL, 8)   = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_WRITE) = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_EXEC) = 0
futex(0x7f9e564b6a04, FUTEX_WAKE_PRIVATE, 1) = 1
futex(0x7f9e5a22cddc, FUTEX_WAKE_PRIVATE, 1) = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\246\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 70, NULL, 8)   = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_WRITE) = 0
mprotect(0x662d2804000, 503808, PROT_READ|PROT_EXEC) = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\247\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 73, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\207\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 74, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\217\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 77, NULL, 8)   = 0
epoll_pwait(3, [], 1024, 0, NULL, 8)    = 0
write(17, "\0337\33[55;1H\33[55;2H\33[47;90m\342\240\213\33[m\0338", 32) = 32
epoll_pwait(3, [], 1024, 66, NULL, 8)   = 0
write(8, "\1\0\0\0\0\0\0\0", 8)         = 8
--- SIGSEGV {si_signo=SIGSEGV, si_code=SEGV_MAPERR, si_addr=NULL} ---
+++ killed by SIGSEGV (core dumped) +++
```

**Comment by astefanutti** [comment]
Thanks a lot for the report. I've been able to reproduce. Let me dig into this ASAP and get back to you.

**Comment by astefanutti** [comment]
This should be fixed with f4b4e1996d44945c800dc3e46813dd229ad5aa67.

I've re-pushed the binary in the 0.6.0 release. Could you please give it a try?

**Comment by malcolm061990** [comment]
Thanks! All works as expected!
